src/App.js

```diff
@@ -1,24 +1,25 @@
-import logo from './logo.svg';
-import './App.css';
+import { Provider } from "react-redux";
 
-function App() {
+import { AddQuestionForm } from "./addQuestion/index.js";
+import { ListOfQuestions } from "./listOfQuestions/index.js";
+import { Score } from "./score/index.js";
+import { CurrentStreak } from "./currentStreak/index.js";
+
+import "./App.css";
+
+function App({ store }) {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Provider store={store}>
```

Comment:
I usually make a `withRedux` component that I add to a page HOC that I compose with every page component. That component also handles cross-cutting concerns such as authentication/authorization, etc.